msteams-platform/m365-apps/prerequisites.md

```diff
@@ -87,10 +87,16 @@ To install Microsoft 365 applications in your test environment:
     |Outlook Add-ins | `BetaChannel` |
     |Meeting apps in Outlook | `Current` |
 
+   | **Word, Excel, PowerPoint scenario** | **Update channel** |
+    |---------------|--------------|
+   |Word, Excel, PowerPoint Add-ins | `BetaChannel` |
+   
+   
+
 1. Open Command Prompt and go to the local folder path.
 1. Run `setup.exe /configure configuration-Office365-x86.xml` (or use the **x64.xml* file, depending on your setup).
-1. Open Outlook (desktop client) and set up the mail account using your test tenant credentials.
-1. Open **File** > **Office Account** > **About Outlook** to confirm you're running a Microsoft 365 *Current Channel* build of Outlook.
+1. Open Outlook, Word, Excel and PowerPoint (desktop client) and set up the mail account using your test tenant credentials.
+1. Open **File** > **Office Account** > **About Outlook** to confirm you're running a Microsoft 365 *Current Channel* build of Outlook. Open **File** > **Account** > **Update Channel** to confirm you're running a Microsoft 365 *Beta Channel* build of Word, Excel, and PowerPoint.
```

Comment:
Appending this sentence makes it sound like it's possible to run different update channels for different Office applications . . . is that true? If not, could we reword this more generally to cover both cases (Outlook GA / WXP preview)? For example:

1. If you're testing in Outlook, open **File** > **Office Account** > **About Outlook** to confirm you're running a Microsoft 365 *Current Channel* build of Outlook. If you're testing in Word, Excel, or PowerPoint, open **File** > **Account** > **Update Channel** to confirm you're running a Microsoft 365 *Beta Channel* build.